lib/game.rb

```diff
@@ -13,8 +16,8 @@ def initialize(row)
     @date_time = row["date_time"]
     @away_team_id = row["away_team_id"]
     @home_team_id = row["home_team_id"]
-    @away_goals = row["away_goals"]
-    @home_goals = row["home_goals"]
+    @away_goals = row["away_goals"].to_i
+    @home_goals = row["home_goals"].to_i
```

Comment:
wise decision to convert to integer here. Saves us time from having to do it each time we call on these instance variables